inverted_index.py

```diff
@@ -58,7 +58,7 @@ def main():
 
     corpus = load_data(data_path)
     stop_words = load_stop_words(stop_words_path)
-
+    stop_words = []
```

Comment:
You want to remove this before merging in master, right ?

Reply:
Well it should eventually be implemented differently, the idea is to not remove stop words from the index.

Reply:
I disagree. I think we want to remove stop words, except the ones that are in one of the 8 queries, right ?

Reply:
The problem with that is that the "search engine" doesn’t work for other queries that contain only stop words.

Reply:
You're right. But we know that it will be evaluated only on the queries 2,3,4,8, and the concatenation of all queries, so we can just remove these words from the stop words... 😈 
Let's try without stop words, if performances are good enough let's keep it like that.
Or we can have two indexes, one with and one without stop words. When a query only has stop words, we compute it on the index with stop words.

Reply:
I had thought of removing stop words from the query only if non stop words are also present in query. But I don’t understand the point of removing stop words from the index, is it just to reduce the index size ?

Reply:
If you remove stop words from the queries, I think the goal is just to reduce index size.
So yes, we can try to leave stop words in the index and remove stop words only in the queries that also have non stop words. It should work.

I am not sure about the performances when building the index with the stop words (it could add a lot of computation). But let's try it.